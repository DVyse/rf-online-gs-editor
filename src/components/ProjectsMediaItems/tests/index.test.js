// import React from 'react';
// import { shallow } from 'enzyme';

// import ProjectsMediaItems from '../index';

describe('<ProjectsMediaItems />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
