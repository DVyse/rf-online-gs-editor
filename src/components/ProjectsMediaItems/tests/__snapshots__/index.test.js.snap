// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectsMediaItems /> Should exist 1`] = `
<ItemGroup
  divided={true}
/>
`;

exports[`<ProjectsMediaItems /> Should render items 1`] = `
<ItemGroup
  divided={true}
>
  <ProjectsMediaItem
    currentUser={null}
    item={
      Object {
        "createdAt": "2018-01-01 06:00",
        "description": "description",
        "id": "123",
        "isPublic": false,
        "name": "name",
        "owner": Object {
          "id": "456",
          "login": "name",
        },
        "title": "title",
      }
    }
    key="123"
  />
</ItemGroup>
`;
